azuki-script-generation/src/main/kotlin/com/anaplan/engineering/azuki/script/generation/ScriptGenerationTestHelper.kt

```diff
@@ -0,0 +1,44 @@
+package com.anaplan.engineering.azuki.script.generation
+
+import com.anaplan.engineering.azuki.core.parser.ScenarioParser
+import com.anaplan.engineering.azuki.core.parser.ScenarioParsingContext
+import com.anaplan.engineering.azuki.core.parser.SimpleScenarioParser
+import com.anaplan.engineering.azuki.core.scenario.BuildableScenario
+import com.anaplan.engineering.azuki.core.system.ActionFactory
+import org.junit.Assert
+import org.slf4j.LoggerFactory
+
+open class ScriptGenerationTestHelper<S: BuildableScenario<AF>, AF: ActionFactory>(
```

Comment:
❓ Better name suggestions would be gratefully received.

Also, should this be `open` or `abstract` (given that it'll invariably be subclassed into an `object`)?